### DISCUSSION TITLE
能否提供一个采用zhipu的glm4调用的例子

### DISCUSSION THREAD

**Discussion by YinSonglin1997** [modelscope/ms-agent#454] [Q&A]
能否提供一个采用zhipu的glm4调用的例子，我想启动前端界面又该如何执行呢？Readme中似乎并没有给出介绍

**Comment by slin000111** [comment]
agentfabric有gradio界面，可以选glm-4。https://github.com/modelscope/modelscope-agent/blob/master/apps/agentfabric/README_CN.md
